.gitignore

```diff
@@ -38,3 +38,5 @@ project/plugins/project/
 # System Files
 .DS_Store
 Thumbs.db
+
+*.sc
```

Comment:
Why did you add `*.sc` here?

Reply:
I use worksheets for prototyping and thought it could be useful if others so to so that they are ignored as they are generally not implementation files.. I could have explicites that's true